src/pages/safe/transactions/tx.tsx

```diff
@@ -50,14 +51,15 @@ const SingleTransaction: NextPage = () => {
   )
 
   const isCurrentSafeTx = sameAddress(txDetails?.safeAddress, safeAddress)
+  const breadcrumbsLink = `${AppRoutes.safe.transactions.index}?safe=${safeAddress}`
```

Comment:
Perhaps we could automate by remove everything after the last `/`?

Reply:
I’d go with this custom link for now, but good idea for another PR.